api/src/main/java/org/openmrs/module/openconceptlab/client/OclClient.java

```diff
@@ -125,8 +131,26 @@ public GetMethod executeExportRequest(String url, String latestVersion) throws I
 		return exportUrlGet;
 	}
 
+	public String  getUrlVersion(String url, String token) throws IOException {
+		String exportVersion;
+
+		if (url.endsWith("/")) {
+			url = url.substring(0, url.lastIndexOf('/'));
+		}
+		String[] words = url.split("api.openconceptlab.org");

```

Comment:
Instead of hard coding `api.openconceptlab.org`, how about putting it in a global property?